vrk-kpa/suomidigi
# SUOPA-98 Event form

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh/form/add-new-event with an incognito mode and fill out the form (completely) and submit it.
- Log in and go to https://suomidigi.docker.sh/admin/content
   - Check that there is a new node of the event what you've just send through the form
   - Edit the node and check that it is unapproved and unpublished
   - Also check that all necessary information has been saved to the node (f.e. The dates, image, multiple organiser contact info, etc..
